tools/generate_events.py

```diff
@@ -81,7 +83,7 @@
 }
 
 func (e *{{eventName}}) GetName() string {
-    return "{{event['name']}}"
+    return e.EventName
```

Comment:
my only comment is we might either delete the `GetName` function and use `e.EventName` directly
or change the `EventName` to be unexported and retain the `GetName`, assuming I haven't missed anything.
Having the `EventName` exported might let the caller the ability to modify the event-name which isn't desired.

Reply:
Changed it to unexported, to `eventName`